app/proguard-rules.pro

```diff
@@ -110,6 +110,14 @@
 -dontwarn com.google.zxing.MultiFormatWriter
 -dontwarn com.google.zxing.common.BitMatrix
 
+################################################################################
+# Androidx Credential Manager
+################################################################################
+-if class androidx.credentials.CredentialManager
+-keep class androidx.credentials.playservices.** {
+  *;
```

Comment:
Is this required because of a bug in the `CredentialManager`?

Reply:
Meant to remove this. It's not needed.